src/views/Projects/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+import { useQuery } from '@apollo/client';
+import getProjects from 'queries/getProjects';
+import { GetProjects } from 'queries/types/GetProjects';
+
+const Projects = () => {
+  const { loading, error, data } = useQuery<GetProjects>(getProjects);
```

Comment:
I don’t love that you have to require both the type definition and query and then pass them into `useQuery`.